### ISSUE TITLE
Add janitor label

### ISSUE THREAD

**Issue by tcharding** [rust-bitcoin/rust-bitcoin#706]
TL;DR - Add new label 'janitor'

- Like nice-to-have but less interesting. 
- Like good-first-issue but more boring / require a bit too much work to suggest new folks pick them up.

Could we please have a new label 'janitor'. This label can then be used for issues that are janitorial in nature, by that I mean boring grunt work that folks would like to see done but can't really be bothered doing themselves. Perhaps things folks have had on their TODO list for ages and not got around to doing. Having the label would allow devs to add these issues without being concerned that they are burdening others, the tasks are by nature non-urgent.

**Comment by Kixunil** [comment]
Sounds a lot like what I call "code quality". Is that the same thing?

**Comment by tcharding** [comment]
'code quality' sounds very much like it. Thinking out loud here; I wonder, is there any tasks that fall outside of 'code quality' that 'janitor' would include, I'm struggling to think of any. Do you think adding a 'code quality' label would be useful @Kixunil?

>  Isn't this sort of things usually named something like "good first time issue"?

I don't believe this is the same, specifically I do not think its good to push 'boring' menial tasks towards first time contributors, especially if they are laborious and/or require some bikeshedding to get them merged.

**Comment by tcharding** [comment]
Here is an example issue that could be raised if we had this label. Without this label I would typically deem it too trivial to raise, I would usually just put this sort of thing on my TODO list and do it myself later. With this label I'm trying to tease out of other developers their tasks like this one.

"Change all function rustdocs to use 3rd person tense"

(based on discussion on this [docs cleanup](https://github.com/rust-bitcoin/rust-bitcoin/pull/704) PR)

**Comment by Kixunil** [comment]
> Do you think adding a 'code quality' label would be useful @Kixunil?

I do, I usually add it to my own projects lately.

>  Without this label I would typically deem it too trivial to raise, I would usually just put this sort of thing on my TODO list and do it myself later.

Sounds like we also want priority labels? (e.g. `P-low`, `P-medium` `P-high`)

**Comment by tcharding** [comment]
> Sounds like we also want priority labels? (e.g. P-low, P-medium P-high)

Yes please!

**Comment by Kixunil** [comment]
Added the labels, hope I'm not overstepping boundaries as a new collaborator. Should be easy to change/remove anyway. :) I suggest to keep this open for a bit if others want to express their opinion.

To make it easier, if at least two collaborators react to this with thumbs up emoji I will close it.

**Comment by tcharding** [comment]
Got two thumbs up, closing.

**Comment by apoelstra** [comment]
ACK all these suggestions
